package.json

```
@@ -11,6 +11,7 @@
     "@types/jest": "24.0.17",
     "@types/node": "12.7.0",
     "@types/react": "16.8.24",
+    "@types/react-bootstrap": "^0.32.20",
```

Can you set it fixed to a concrete version? (I mean remove the `^`)

```suggestion
    "@types/react-bootstrap": "0.32.20",
```

---

Fixed it! 44d1b16